akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/Effects.scala

```diff
@@ -4,7 +4,7 @@
 
 package akka.actor.testkit.typed.scaladsl
 
-import akka.actor.typed.{ ActorRef, Behavior, Props }
+import akka.actor.typed.{ActorRef, Behavior, Props}
```

Comment:
Can we remove that setting? Seems like a useless change that just might get us more conflicts.

Reply:
+1, the current formatting is nice, it ads that padding

Reply:
fixed: `spaces.inImportCurlyBraces = true`